### PR TITLE
Removing duplicate grover test

### PR DESCRIPTION
**Pull request by gusty1g**
# Pull Request Template

## Description

This PR removes a duplicate test for GroverEmbedding. The same test is also present here: 
https://github.com/deepchem/deepchem/blob/96050f8b4808a6b50d1eaf47759e405fe232544e/deepchem/models/torch_models/tests/test_grover_layers.py#L26

## Type of change

Please check the option that is related to your PR.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
  - In this case, we recommend to discuss your modification on GitHub issues before creating the PR
- [ ] Documentations (modification for documents)

## Checklist

- [x] My code follows [the style guidelines of this 
project](https://deepchem.readthedocs.io/en/latest/development_guide/coding.html)
  - [x] Run `yapf -i <modified file>` and check no errors (**yapf version must be  0.32.0**)
  - [x] Run `mypy -p deepchem` and check no errors
  - [x] Run `flake8 <modified file> --count` and check no errors
  - [x] Run `python -m doctest <modified file>` and check no errors
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New unit tests pass locally with my changes
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: deepchem/models/torch_models/tests/test_grover.py [hunk 1]
```diff
@@ -1,27 +0,0 @@
...
-    pass
-
-
-def testGroverEmbedding():
-    from deepchem.models.torch_models.grover_layers import GroverEmbedding
-    hidden_size = 8
-    f_atoms = torch.randn(4, 151)
-    f_bonds = torch.randn(5, 165)
-    a2b = torch.Tensor([[0, 0], [2, 0], [4, 0], [1, 3]]).type(torch.int32)
-    b2a = torch.Tensor([0, 1, 3, 2, 3]).type(torch.long)
```
Comment (rbharath) [inline deepchem/models/torch_models/tests/test_grover.py:13]: This test looks slightly longer than the duplicate test. Are there any meaningful difference between this test and the one in test_grover_layers?

**Review by rbharath — COMMENTED** [review]
A quick clarification question

File: deepchem/models/torch_models/tests/test_grover.py (hunk 1 above)
Comment (gusty1g) [reply deepchem/models/torch_models/tests/test_grover.py:13]: The values in `test_grover` are randomly initialized while the values in `test_grover_layers` are from featurized data, so we don't have random initialization there.

**Review by rbharath — APPROVED** [review]
LGTM, feel free to merge in once CI is clear

**Pull request merged**